.github/workflows/ci.yml

```diff
@@ -0,0 +1,25 @@
+name: ci
```

Comment:
You should be able to just remove this file; quay should auto-build when you push to 3.x.

Reply:
As discussed in Slack, this is not really pushing just building the image to get quick feedback on whether the image is building right or not.